data_entry/how-to-enter-recency-data.md

```diff
@@ -0,0 +1,23 @@
+## How To Enter Recency Data Within The Hts Client Card
```

Comment:
@ODORA0 Add a link to the summary page

Reply:
@ssmusoke Which summary page is this, am not sure I got what you requesting?